os.go

```diff
@@ -16,6 +17,9 @@ const assetPrefix string = "/$asset$/"
 
 var fs afero.Fs
 
+// ErrFileNotExist caused by file not found and http get failed
+var ErrFileNotExist = errors.New("FileNotExist")
```

Comment:
Should use same naming as afero/os uses:
https://godoc.org/github.com/spf13/afero#pkg-variables

So this should be 
```
var ErrFileNotFound = os.ErrNotExist
```

Reply:
I change the error to `os.ErrNotExist`, and add two tests